### PR TITLE
3.1.1 release

### PR DESCRIPTION
**Pull request by programmatix**

### CODE REVIEW

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -29,11 +29,55 @@ Java 11 has various enhancements like lambda local variable type inference, prof
...
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.1.1/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:3.1.1**
+* com.couchbase.client:**core-io:2.1.1**
+* io.projectreactor:**reactor-core:3.4.1**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+== API Affecting
```
Comment (daschl) [inline modules/project-docs/pages/sdk-release-notes.adoc]: this has a different heading type than the others

File: modules/howtos/pages/n1ql-queries-with-sdk.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 :page-topic-type: howto
 :page-aliases: n1ql-query,ROOT:querying-n1ql
 :lang: Java
-:version: 3.1.0
+:version: 3.1.1
```
Comment (RichardSmedley) [inline modules/howtos/pages/n1ql-queries-with-sdk.adoc:6]: Technical debt - we should be using the attributes.adoc file in project-docs to pull this in.
If we even need it in the doc?

File: modules/project-docs/pages/sdk-release-notes.adoc (hunk 1 above)
Comment (programmatix) [reply modules/project-docs/pages/sdk-release-notes.adoc]: ooh good spot

**Review by RichardSmedley — APPROVED** [review]

File: modules/howtos/pages/n1ql-queries-with-sdk.adoc (hunk 2 above)
Comment (programmatix) [reply modules/howtos/pages/n1ql-queries-with-sdk.adoc:6]: I thought you added this field :D  It drives me nuts changing the version numbers everywhere, if there's a better way to do it then yes please!

Comment (RichardSmedley) [reply modules/howtos/pages/n1ql-queries-with-sdk.adoc:6]: Track fix at DOC-7969

**Pull request merged**